lib/builders/latexmk.js

```diff
@@ -62,10 +62,20 @@ export default class LatexmkBuilder extends Builder {
     const enableShellEscape = atom.config.get('latex.enableShellEscape')
     const enableSynctex = atom.config.get('latex.enableSynctex') !== false
     const engine = this.getLatexEngine(filePath)
+    const cleanExtensions = atom.config.get('latex.cleanExtensions')
 
-    if (shouldRebuild) {
-      args.push('-g')
+    switch (action) {
+      case 'rebuild':
+        args.push('-g')
+        break
+      case 'clean':
+        args.push('-c')
+        if (cleanExtensions.length) {
+          args.push(`-e "\\$clean_ext='${cleanExtensions.join(' ')}'"`)
+        }
+        break
```

Comment:
Yeah, I realize that, but I'm just questioning whether it makes sense. Also `-gg` is not strictly equivalent to what we're supporting now. What we have now is a "normal" clean (can be configured to be equivalent to `-gg` by the user) which can then be followed by a build. If the user wants to control when the do a `-c` style clean (our current default) they can. With the new scheme, they don't have that control anymore.

I'm not saying having this functionality is vital or even makes sense, I'm simply arguing the fact we are changing the current behavior. We reduce code complexity somewhat (if we ignore the magic `action` strings), but we also lose some control and flexibility.